tests/keras/legacy/interface_test.py

```diff
@@ -59,5 +60,12 @@ def test_avgpooling1d_legacy_interface():
     assert json.dumps(old_layer.get_config()) == json.dumps(new_layer.get_config())
 
 
+@keras_test
+def test_prelu_legacy_interface():
+    old_layer = keras.layers.PReLU(init='zero', weights=zeros(5), name='p')
```

Comment:
Yeah, I noticed they went back to the Layer class, but put it in for my own check to ensure it was still working. I'll take it out.  I'm keeping the `name` argument in since I saw that it failed when I took it out.